src/ui/library/InputLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import globalStyles from '@ui/styles';
```

Comment:
Let's create the necessary styles here and not import global styles to make the library components as independent as possible.